packages/ui/src/components/CookieBanner/CookieBanner.tsx

```diff
@@ -32,32 +31,25 @@ export const CookieBanner = (props: CookieBannerProps) => {
       bg="gray.700"
       rounded={'md'}
       boxShadow="sm"
-      {...rest}
+      position="fixed"
+      bottom="0"
+      right="0"
+      left="0"
     >
-      {onClose && (
-        <CloseButton
-          display={{ sm: 'none' }}
-          position="absolute"
-          right="2"
-          top="2"
-          color={'lightgray'}
-          onClick={onClose}
-        />
-      )}
-
       {!isMobile && (
         <Square size="12" bg="lightgray" borderRadius="md">
```

Comment:
You can remove bg color

```suggestion
        <Square size="12" borderRadius="md">
```